pkg/controller/migmigration/backup.go

```diff
@@ -255,3 +256,154 @@ func (t *Task) bounceResticPod() error {
 
 	return nil
 }
+
+// Annotate all resources with PV action data
+func (t *Task) annotateStorageResources() error {
+	// Get client of source cluster
+	client, err := t.getSourceClient()
+	if err != nil {
+		return err
+	}
+	namespaces := t.PlanResources.MigPlan.Spec.Namespaces
+	pvs := t.PlanResources.MigPlan.Spec.PersistentVolumes
+	for _, pv := range pvs.List {
+		// Update PVs with their action
+		resource := corev1.PersistentVolume{}
+		err := client.Get(
+			context.TODO(),
+			types.NamespacedName{
+				Name: pv.Name,
+			},
+			&resource)
+		if err != nil {
+			return err
+		}
+		if resource.Annotations == nil {
+			resource.Annotations = make(map[string]string)
+		}
+		resource.Annotations[pvAnnotationKey] = pv.Action
+		if resource.Labels == nil {
+			resource.Labels = make(map[string]string)
+		}
+		resource.Labels[pvBackupLabelKey] = pvBackupLabelValue
```

Comment:
To answer myself:

```
We are querying for this const label before removing any PV action annotations:
openshift.io/pv-backup: transient
```

Reply:
Yup, this label allows for us to find every resource we have annotated so that we can remove them after a successful backup. This allows us to run multiple migrations on the same namespace (mainly for testing/development)